pynicotine/plugins/core_commands/__init__.py

```diff
@@ -152,8 +152,8 @@ def __init__(self, *args, **kwargs):
                 "description": _("Browse files of user"),
                 "disable": ["cli"],
                 "group": _CommandGroup.USERS,
-                "usage": ["<user>"],
-                "usage_private_chat": ["[user]"]
+                "usage": ["<user\\folder or url>"],
+                "usage_private_chat": ["[user\\folder]", "[url]"]
```

Comment:
Is this syntax based on some existing convention? Not sure I like it, to me it reads like two positional arguments.

Reply:
Agreed, I shouldn't have put it in the Network Filter commands either. It does look confusing.